src/utils/constants.ts

```diff
@@ -11,3 +11,6 @@ export const containsSpecialCharRegex = /[@$!%*#?&]/;
 
 // cooldown time to save deer in ms (1000ms = 1s)
 export const cooldownTime = 60000;
+
+// cooldown for ever box highlight
+export const detectCooldown = 5000;
```

Comment:
Just a thought, don't you think for the future we can make this a variable that is stored in our db or Azure (or aws) where we can update it without needing to go through a full release? 
For now, this is fine, but for continued development, we will want a way to update such important variables on the fly.
Lets put in a story for this

Reply:
good call, ill make that a story from right now and have the detection cooldown be something fetched from AWS/Azure in the future